src/AddCommand.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class AddCommand implements Command {
+    private final UsersBook usersBook;
+    private final String SEPARATOR = "____________________________________________________________________________";
```

Comment:
SEPARATOR is constant ?  idea don't notify you that this variable can be better ?
I think that SEPARATOR is common for all Commands. Maybe you can move  it to some common for all commands place